imjoy/imjoyUtils.py

```diff
@@ -153,12 +152,11 @@ def task_worker(self, q, logger, abort):
                             result = method(*args)
                             resolve(result)
                         except Exception as e:
+                            traceback_error = traceback.format_exc()
                             logger.error(
```

Comment:
If we want to log the traceback, we can use `logger.exception`.

Reply:
Sounds good. And one thing we need to think is also how do we actually use these logs.

Ideally, in the future, we should have a way to collect these logs and expose to the ImJoy app, and each plugin, and also the engine should have a place to read these logs, and depending on different types, we can apply css styles (e.g.: error in red) to show a log window.